### PR TITLE
Update release notes based on feedback from minimalism session.

### PR DESCRIPTION
**Pull request by michellemacrh**
Update the following files:
- platform-intro.adoc
- aap-24.adoc
- controller-440.adoc
- eda-24.adoc

Add the following snippets:
- deprecated-features.adoc
- technology-preview.adoc

Also add a symlink to the snippets directory.

### CODE REVIEW

File: downstream/snippets/deprecated-features.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+Deprecated functionality is still included in {PlatformNameShort} and continues to be supported. However, it will be removed in a future release of this product and is not recommended for new deployments.
```
Comment (sayjadha) [inline downstream/snippets/deprecated-features.adoc:1]: Suggestions for clarity:

- 'it' -> 'the functionality'
- 'this product -> platform name or 'the platform'

File: downstream/snippets/technology-preview.adoc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```
Comment (sayjadha) [inline downstream/snippets/technology-preview.adoc:1]: This is a recommendation, so feel free to use/refuse as you deem fit:
I'd prefer having the 3rd statement at the start of the paragraph, and the current 1st and 2nd statement to follow right after. Reason being: Sounds better if we start with what is a Technology preview feature, market it as an exciting opportunity to try out upcoming feature, and then state the disclaimer that these features aren't supported by RH SLAs.

File: downstream/titles/release-notes/docinfo.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <productnumber>2.4</productnumber>
 <subtitle>New features, enhancements, and bug fix information</subtitle>
 <abstract>
-    <para>This guide provides a summary of new features, enhancements, and bug fix information for Red Hat Ansible Automation Platform.</para>
+    <para>The release notes for Red Hat Ansible Automation Platform summarize all new features and enhancements, notable technical changes, major corrections from the previous version, and any known bugs upon general availability.</para>
```
Comment (sayjadha) [inline downstream/titles/release-notes/docinfo.xml:6]: Suggest 'Red Hat AAP'-> Attribute **{PlatformName}**

File: downstream/titles/release-notes/topics/aap-24.adoc [hunk 4]
```diff
@@ -3,13 +3,36 @@
...
+* For information about execution node enhancements on OpenShift deployments, see link:https://docs.ansible.com/automation-controller/latest/html/administration/instances.html[Managing Capacity With Instances].
 
-{PlatformName} simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across many IT domains including operations, networking, security, development, and across diverse hybrid environments. Simple to adopt, use, and understand, {PlatformName} provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
 
 == New features and enhancements
 
-This release of {PlatformNameShort} features the following enhancements:
+{PlatformNameShort} 2.4 includes the following enhancements:
 
 * Before this update, the {ExecEnvShort} container images were based on RHEL 8 only. With {PlatformNameShort} 2.4 onwards, the {ExecEnvShort} container images are now also available on RHEL 9.
```
Comment (sayjadha) [inline downstream/titles/release-notes/topics/aap-24.adoc:37]: Suggest 'Before this update' -> 'Previously' or 'Earlier' for clarity.

File: downstream/titles/release-notes/topics/aap-24.adoc [hunk 5]
```diff
@@ -86,37 +110,10 @@ The following bugs were fixed in this release of {PlatformNameShort}:
 
 * Added a FQDN check for the {HubName} hostname required to run the Skopeo commands.
 
-* Fixed an issue such that the front end URL for Red Hat Single Sign On (SSO) is now properly configured after you specify the `sso_redirect_host` variable.
+* The front end URL for Red Hat Single Sign On (SSO) is now properly configured after you specify the `sso_redirect_host` variable.
```
Comment (sayjadha) [inline downstream/titles/release-notes/topics/aap-24.adoc:113]: This list item doesn't conform with grammatical parallelism with rest of the list points. Looks like almost all start with a verb in past tense, so the earlier construction was better suited.

**Comment by sayjadha** [comment]
Some minor feedback for your review and incorporation, @michellemacrh . Rest looks very well done!

File: downstream/snippets/technology-preview.adoc (hunk 2 above)
Comment (michellemacrh) [reply downstream/snippets/technology-preview.adoc:1]: Completely agree, thank you for the feedback :)

**Review by sayjadha — COMMENTED** [review]
LGTM.

**Pull request merged**